chore(macros): Remove SectionOnPage macro usage (#23059)
mdn/content: files/en-us/web/api/rtcpeerconnection/signalingstate/index.md

describing the state of the signaling process
on the local end of the connection
while connecting or reconnecting to another peer.

> See the [Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) section for more details about the process.

Reviewer: ```suggestion
See [Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) for more details about the process.
```

(Or "See \[Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) in our WebRTC session lifetime guide")

> See the [Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) section for more details about the process.

Author: ```suggestion
See [Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) in our WebRTC session lifetime page.
```

> See [Lifetime of a WebRTC session, Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) for more details about the signaling process.

Reviewer 2: ```suggestion
See the [Signaling](/en-US/docs/Web/API/WebRTC_API/Session_lifetime#signaling) section for more details about the process.
```
